week2/materials/homework/assignments.md

```diff
@@ -4,27 +4,28 @@
 
 Sync your changes to s3 and write down your s3 bucket url.
 
-bucket url: `?`
+bucket url: `https://s3.console.aws.amazon.com/s3/buckets/soujanya-week2-homework?region=us-east-1&tab=objects`
 
 **Assignment 5:**
 
 Right now, the website does not support https. Explain which other AWS service needs to be integrated in order to achieve this.
 
-service name: `?`
+service name: `AWS Certificate Manager`
 
 **Assignment 6:**
 
 Figure out and write down the price per month of storing 51TB on S3.
 
-Price: `?`
+Price: `First 50 TB / Month:	$0.023 per GB`
```

Comment:
We are looking for the total cost pr. month for 51TB data storage not the price pr. gigabyte. you need to calcuate the total cost

Reply:
Ohhh ok @bytewiz